src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -0,0 +1,569 @@
+            setUpComposeFocusManager();
+        });
+        KeyDownListener.addKeyDownPressListner(focusComposerOnKeyPress);
+
+        setUpComposeFocusManager();
+
+        return () => {
+            ReportActionComposeFocusManager.clear(true);
+
+            KeyDownListener.removeKeyDownPressListner(focusComposerOnKeyPress);
+            unsubscribeNavigationBlur();
+            unsubscribeNavigationFocus();
+        };
+    }, [focusComposerOnKeyPress, navigation, setUpComposeFocusManager]);
+
+    const prevIsModalVisible = usePrevious(modal.isVisible);
+    const prevIsFocused = usePrevious(isFocused);
+    useEffect(() => {
+        if (modal.isVisible && !prevIsModalVisible) {
+            // eslint-disable-next-line no-param-reassign
+            isNextModalWillOpenRef.current = false;
+        }
+        // We want to focus or refocus the input when a modal has been closed or the underlying screen is refocused.
+        // We avoid doing this on native platforms since the software keyboard popping
+        // open creates a jarring and broken UX.
+        if (!(willBlurTextInputOnTapOutside && !isNextModalWillOpenRef.current && !modal.isVisible && isFocused && (prevIsModalVisible || !prevIsFocused))) {
+            return;
+        }
+
+        focus();
+    }, [focus, prevIsFocused, prevIsModalVisible, isFocused, modal.isVisible, isNextModalWillOpenRef]);
+
+    useEffect(() => {
+        if (value.length === 0) {
+            return;
+        }
+        Report.setReportWithDraft(reportID, true);
+
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    useImperativeHandle(
+        forwardedRef,
+        () => ({
+            blur,
+            focus,
+            replaceSelectionWithText,
+            prepareCommentAndResetComposer,
+            isFocused: () => textInputRef.current.isFocused(),
+        }),
+        [blur, focus, prepareCommentAndResetComposer, replaceSelectionWithText],
+    );
+
+    return (
+        <>
+            <View style={[containerComposeStyles, styles.textInputComposeBorder]}>
+                <Composer
+                    checkComposerVisibility={checkComposerVisibility}
+                    autoFocus={shouldAutoFocus}
+                    multiline
+                    ref={setTextInputRef}
+                    textAlignVertical="top"
+                    placeholder={inputPlaceholder}
+                    placeholderTextColor={themeColors.placeholderText}
+                    onChangeText={(commentValue) => updateComment(commentValue, true)}
+                    onKeyPress={triggerHotkeyActions}
+                    style={[styles.textInputCompose, isComposerFullSize ? styles.textInputFullCompose : styles.flex4]}
+                    maxLines={maxComposerLines}
+                    onFocus={onFocus}
+                    onBlur={onBlur}
+                    onClick={setShouldBlockSuggestionCalc}
```

Comment:
Is this intended? Previously we used to set  `shouldBlockEmojiCalc` and `shouldBlockMentionCalc` to `false` but now this being set to `true`.

Reply:
I dont think there should be change in the behaviour here, not intentionally